xds/src/test/java/io/grpc/xds/internal/security/certprovider/FileWatcherCertificateProviderProviderTest.java

```diff
@@ -85,6 +109,32 @@ public void createProvider_minimalConfig() throws IOException {
             eq("/var/run/gke-spiffe/certs/certificates.pem"),
             eq("/var/run/gke-spiffe/certs/private_key.pem"),
             eq("/var/run/gke-spiffe/certs/ca_certificates.pem"),
+            eq(null),
+            eq(600L),
+            eq(mockService),
+            eq(timeProvider));
+  }
+
+  @Test
+  public void createProvider_minimalSpiffeConfig() throws IOException {
+    if (!enableSpiffe) {
```

Comment:
Use `Assume`. `Assume.assumeTrue(enableSpiffe)`. It will report that the test was skipped, instead of showing the test as green. (It doesn't cause a failure, the test will be grey typically.)